src/lib/components/event/event-summary-row.svelte

```diff
@@ -263,17 +263,18 @@
 </script>
 
 <tr
-  class="hover:cursor-pointer"
-  class:border={failure || canceled || terminated}
-  class:border-danger={failure}
-  class:border-warning={canceled}
-  class:border-pink-700={terminated}
+  class="hover:cursor-pointer {failure &&
+    '!bg-red-400/40 hover:!bg-red-400/60'} {canceled &&
+    '!bg-yellow-400/30 hover:!bg-yellow-400/50'} {terminated &&
+    '!bg-pink-700/30 hover:!bg-pink-700/50'} {hasRelatedActivities(
+    group,
+    hoveredEventId,
+  ) && 'active'}"
```

Comment:
Seems like the perfect place for `twMerge`. We could also use `bg-danger` and `bg-warning` here I think. 

```suggestion
  class={merge(
    failure && '!bg-danger hover:!bg-danger/60',
    canceled && '!bg-warning hover:!bg-warning/60',
    terminated && '!bg-pink-700/30 hover:!bg-pink-700/60',
    hasRelatedActivities(group, hoveredEventId) && 'active',
  )}
```

Reply:
Only bummer is our custom classes like bg-danger/warning don't work with opacity. Gonna keep the custom colors because I want to have a slight change in opacity on hover. Did pull in merge though

Reply:
I tested with opacity (e.g. `bg-danger/60`) and it appeared to work. The color was changing on hover 🤔